### DISCUSSION TITLE
How can I get `tab` note SVG and change the color without Re-rendering

### DISCUSSION THREAD

**Discussion by popochess** [opensheetmusicdisplay/opensheetmusicdisplay#1452] [Q&A] [answered]
following the [spec](https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/wiki/Exploring-the-Demo),
the current implementation offers a method for retrieving voice/piano notes through the use of getSVGGElement(). However, I have encountered difficulty in locating a method to access notes in the tablature.

Is there an alternative approach I might get tab notes and change the note color?

👇 The following outcome is the result of my implementation using the OSMD framework. thanks the team!

https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/assets/236130/44d23d56-515a-48cd-b1d9-795d3990ed81

**Comment by popochess** [comment, marked as answer]
I finally figured it out 😂

![demo_cursor](https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/assets/236130/e04dd278-b275-49fd-a8a6-8c554f09a47d)

**Reply by sschmidTU** [reply]
@popochess Can you share the code of how you got the tab SVG? Might be helpful for others. I'm guessing you got the (Vexflow) SVG id. By the way, we should fix getSVGGElement() for tabs in the future.
